.travis.yml

```diff
@@ -21,6 +21,7 @@ addons:
 virtualenv:
   system_site_packages: true
 script:
+- cd proximal/halide && meson build && ninja -C build
 - bash continuous_integration/test_script.sh
```

Comment:
The one-liner change to validate that all C++ build targets can build on linux-x86_64, without CUDA.

Reply:
Awesome, thanks! Yeah, I want to switch to github actions. The CI here is simpler than cvxpy so it will be a good learning experience.

Reply:
> Yeah, I want to switch to github actions. 

I see. I will put on hold here until you made the transition to GH actions. And then I will retract and resubmit this PR.

Reply:
Ok I tried to switch things over to github actions, got pretty far but ran into compilation issues. If you're able to take a look at all I'd really appreciate it!

Reply:
@SteveDiamond I can take a look at it. Could you please send me the link to a (draft) PR?

Reply:
The new CI is on master, in .github/workflows/python-package.yml